dali/base/dadfs.cpp

```diff
@@ -4704,6 +4705,20 @@ protected: friend class CDistributedFilePart;
         else
             return false;
     }
+    virtual double getCost(const char * cluster) override
+    {
+        VStringBuffer path("storage/planes[@name='%s']", cluster);
+        Owned<IPropertyTree> plane = queryGlobalConfig().getPropTree(path);
+        if (!plane)
+            return 0.0;
+        double storageCostDaily = plane->getPropReal("cost/@storageAtRest", 0.0) * 12 / 365;
+
+        CDateTime dt;
+        getModificationTime(dt);
+        double fileAgeDays = difftime(time(nullptr), dt.getSimple())/(24*60*60);
```

Comment:
I can make this function more efficient by passing in the cost rate and the current time.  However, this function is not called very frequently.  This can be improved in the later versions.

Reply:
yes, I would ignore for now